etc/kube/start-kube-docker.sh

```diff
@@ -13,14 +13,6 @@ docker run \
     --net=host \
     --pid=host \
     --privileged=true \
-    gcr.io/google_containers/hyperkube:v1.5.1 \
-    /hyperkube kubelet \
-        --containerized \
-        --hostname-override="127.0.0.1" \
-        --address="0.0.0.0" \
-        --api-servers=http://localhost:8080 \
-        --cluster_dns=10.0.0.10 \
-        --cluster_domain=cluster.local \
-        --config=/etc/kubernetes/manifests \
-        --allow-privileged=true
+    gcr.io/google_containers/hyperkube:v1.6.2 \
+    sh "/rootfs/$PWD/etc/kube/internal.sh"
```

Comment:
Interesting ... it seems like you're running this as a docker command and poking the command through to the root filesystem to get at the script above. Seems like this `docker run ...` one liner is getting complicated ... would it be easier to just make a docker image that contains the script to startup k8s?